src/main/java/org/folio/service/orders/flows/update/open/OpenCompositeOrderFlowValidator.java

```diff
@@ -219,4 +259,14 @@ private Future<List<String>> getPermanentLocationIdsFromHoldings(List<String> ho
       });
   }
 
+  private static String getTenantLocation(String tenantId, String locationId) {
+    return tenantId + "." + locationId;
+  }
+
+  private static RequestContext createContextWithNewTenantId(RequestContext requestContext, String tenantId) {
```

Comment:
regarding places of method. I suggest to move this method below of method which is invoked it

Reply:
e.g
```
getTenants(){
getUsers()
}
getUsers(){}
```

Reply:
This is the way I always follow. When method calls 3 different private methods then I place that 3 methods below in the order caller calls them:
```
public m1() {
  a();
  b();
  c();
}
private a() {}
private b() {}
private c() {}
```
https://stackoverflow.com/questions/4668218/are-there-any-java-method-ordering-conventions

Reply:
Yes, I understand your intention and it makes sense for getTenantLocations method. 

for first look I didn't notice createContextWithNewTenantId method used many places. so asked to move where it is invoked. 

Btw, I checked it is actually not used, we can remove

Reply:
Thanks, the usage is there now. I was in the process of fixing unit tests. Please review again

Reply:
From it is purpose, method doesn't seem belong to this class, I suggest to move to HelperUtils